accounts/tests/serializers/test_user_create.py

```diff
@@ -0,0 +1,45 @@
+from django.test.testcases import TestCase
+from accounts.serializers import UserCreateSerializer
+from unittest.mock import patch
+from firebase_admin import auth
+from rest_framework.serializers import ValidationError
+
+
+class UserCreateSerializerTestCase(TestCase):
+    def setUp(self):
+        self.data = {"email": "test@asd.com"}
+
+    @patch("firebase_admin.auth.create_user")
+    def test_user_create(self, mock_create):
```

Comment:
since we're overriding the behaviour of the create method, can we verify that we still create the django user object?